action.yml

```diff
@@ -23,6 +23,23 @@ description: |
           ecr_repository: 'demo/example'
           dotnet-version: '6.0.x'
   ```
+
+  ## Locating Packages
+
+  To locate Dotnet packages navigate to the link below.
+
+  ```text
+  https://github.com/orgs/variant-inc/packages?ecosystem=all
```

Comment:
What if my repository is not in variant-inc?

Reply:
added some instructions with link, changed to <organization> instead of variant-inc